with-proxy/scripts/ledgersmb_config/config

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/with-contenv /bin/bash
```

Comment:
@walidmujahid Would it make sense to split `start.sh` into two? `config.sh` and `fire-up.sh`, which will then separately callable by the S6 overlay? I think that removes the necessity to reproduce the content of `start.sh` below.